pandas/io/formats/format.py

```diff
@@ -1230,7 +1230,7 @@ def _format(x):
                     if x is None:
                         return "None"
                     elif x is NA:
-                        return "NA"
+                        return formatter(x)
```

Comment:
This means that the `formatter` function needs to be able to handle NAs, which now is maybe not the case?

Reply:
Will do a PR for this, I think this breaks geopandas